packages/bsky/src/db/migrations/20230309T045948368Z-init.ts

```diff
@@ -199,6 +221,7 @@ export async function up(db: Kysely<unknown>): Promise<void> {
     .addColumn('did', 'varchar', (col) => col.primaryKey())
     .addColumn('handle', 'varchar', (col) => col.unique())
     .addColumn('indexedAt', 'varchar', (col) => col.notNull())
+    .addColumn('commitDataCid', 'varchar')
```

Comment:
possibility we want this on a different table since it'll be much higher churn than the rest of the table 🤔

Reply:
I peeled this off, tracking additional sync state for the actor:
 - Commit cid so that we can track what commit they should be at, potentially reindex their current state from scratch or perform debugging.
 - Tally of total too-big commits and rebases, to get a sense for if someone is performing a ton of these operations.